config/release/install.yaml

```diff
@@ -11335,6 +11335,14 @@ spec:
           value: /.sigstore
         image: ghcr.io/kyverno/kyverno:latest
         imagePullPolicy: Always
+        startupProbe:
+          httpGet:
+            path: /health/liveness
+            port: 9443
+            scheme: HTTPS
+          failureThreshold: 12
+          initialDelaySeconds: 30
+          periodSeconds: 10
```

Comment:
a `startupProbe` should have little or no `initialDelaySeconds` and small `periodSeconds` (within reason of your timeout however) for fastest startup times.

Let's say your timeout is set to `3`. You'd want to set `periodSeconds` to `4` or `5` and then `(old_initial + (old threshold * old period)) / new_periodSeconds = new_failureThreshold`

With `periodSeconds` set to `10` you're facing a maximum of `9` seconds of wasted time (plus the `30` currently set for initial.)